lib/gui/models/store.js

```diff
@@ -46,6 +48,11 @@ const DEFAULT_STATE = Immutable.fromJS({
     unmountOnSuccess: true,
     validateWriteOnSuccess: true,
     sleepUpdateCheck: false,
+
+    // Only set this by default if the installed
+    // version is not a production version
+    includeUnstableUpdateChannel: release.getReleaseType(packageJSON.version) !== release.RELEASE_TYPE.PRODUCTION,
```

Comment:
```
                               Production
                                   ^
                                   |
            Unstable, Production   |  Stable, Production
             (e.g. 1.0.0-rc.4)     |   (e.g. 1.0.0)
                                   |
Unstable <-------------------------+----------------------> Stable
                                   |
            Unstable, Snapshot     |  Stable, Snapshot
         (e.g. 1.0.0-rc.4+ad4268b) |  (e.g. 1.0.0+814ba4e)
                                   |
                                   v
                                Snapshot
```

This is just for you @jviotti , I'm not suggesting we add this to the documentation :-D